sys-kernel/linux-firmware/linux-firmware-20240312.ebuild

```diff
@@ -351,7 +358,13 @@ src_install() {
 
 	popd &>/dev/null || die
 
-	if use initramfs ; then
+	# Instruct Dracut on whether or not we want the microcode in initramfs
+	(
+		insinto /usr/lib/dracut/dracut.conf.d
+		newins - 10-${PN}.conf <<<"early_microcode=\"$(usex initramfs)\""
```

Comment:
It's only ever going to be yes/no, I'd drop the quotes+escapes personally.

Reply:
Done!